From ce34738107678cb76e3da34cd16dc9a34d8dd242 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <jacmet@sunsite.dk>
Date: Mon, 18 Jun 2012 23:12:05 +0200
Subject: [PATCH] configure.ac: fix bashism

'==' is a bashism, breaking configure on systems where /bin/sh isn't bash
(E.G. Ubuntu).  Fixes:

http://autobuild.buildroot.net/results/c3d0d04f8b3df63b3206cecf54dfdd67a4c7ccfb/build-end.log

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 3ab6667..c3b907b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -176,7 +176,7 @@ dnl Checks for library functions.
 AC_CHECK_FUNCS(getopt_long)
 
 CONDITIONAL_SOURCES="device-${arch}.${OBJEXT} ${CONDITIONAL_SOURCES}"
-if test x${arch} == xlinux ; then
+if test x${arch} = xlinux ; then
 	CONDITIONAL_SOURCES="privsep-${arch}.${OBJEXT} ${CONDITIONAL_SOURCES}"
 	AC_DEFINE(USE_PRIVSEP, 1, [Use privsep])
 fi
-- 
1.7.10

